_data/developer.yml

```
@@ -185,6 +185,14 @@ websites:
       - totp
     doc: https://help.compose.com/docs/two-factor-authentication
 
+  - name: Contentstack
+    url: https://www.contentstack.com/
+    img: contentstack.svg
+    tfa:
+      - totp
```

```suggestion
      - proprietary
```
This website does not seem to use the regular `totp` style token generation, but the proprietary Authy method instead.